apparmor.d/groups/virt/cni

```diff
@@ -0,0 +1,35 @@
+abi <abi/3.0>,
```

Comment:
* Add the header.
* You need a file by profile. named `cni-loopback`, `cni-portmap`...